common/djangoapps/util/testing.py

```diff
@@ -31,8 +31,14 @@ class UrlResetMixin(object):
 
     URLCONF_MODULES = None
 
-    def _reset_urls(self, urlconf_modules):
+    def reset_urls(self, urlconf_modules=None):
         """Reset `urls.py` for a set of Django apps."""
+
+        if urlconf_modules is None:
```

Comment:
In general, I prefer to explicitly check for `None` when looking at default arguments. For instance, if this method were passed `[]` for the list of `urlconf_modules`, I'd want it to use that, rather than the default set of `urlconf_modules`.